lib/processors/nonAsciiEscaper.js

```diff
@@ -92,12 +92,16 @@ const encodingMap = {
  * specific encoding name such as "utf8" or "latin1". Simplifies usage of nonAsciiEscaper encoding
  * option such that it can be used standalone without the respective task (e.g. in Splitter, Bundler and related projects).
  *
+ * @public
+ * @alias module:@ui5/builder.processors.nonAsciiEscaper․getEncodingFromAlias
```

Comment:
This is a special character (not a normal dot), as otherwise the template does not render the function.

Reply:
LFTM

Reply:
Not sure what I had in mind with the 'F': funny? fancy? forbidden?

Reply:
"Living for the Moment" aka YOLO?